web/client/plugins/Home.jsx

```diff
@@ -72,7 +73,7 @@ export default {
             name: 'home',
             position: 4,
             tool: connect((state) => ({
-                hidden: sidebarIsActiveSelector(state),
+                hidden: pathnameSelector(state).indexOf("viewer") !== -1, // this is temporary and will have to change when we align also geostory to have sidebar
```

Comment:
This type of connected configuration should be avoid in particular when we take advantage of container priorities. We could apply high priority to Sidebar container so in case a plugin container has also OmniBar the former will be selected and this will solve the case of map viewer and context viewer. Then we can use the override property in localConfig.json to change priority for Login and Home only in the dashboard container. Here there is a [PR](https://github.com/MV88/MapStore2/pull/6) with a fix for this branch, please check it and merge if this approach could work also for you.

The best approach would be to remove omnibar from the map viewer and give high priority to omnibar container but this will require changes on style and context workflow